src/components/markdown/examples/markdown-footnotes.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, h } from '@stencil/core';
+
+const markdown = `
+Here is a simple footnote[^1].
+
+A footnote can also have multiple lines[^2].
+
+You can also use words, to fit your writing style more closely[^note].
+
+[^1]: My reference.
+[^2]: Every new line should be prefixed with 2 spaces.
+  This allows you to have a footnote with multiple lines.
+[^note]:
+    Named footnotes will still render with numbers instead of the text but allow easier identification and linking.
+    This footnote also has been made with a different syntax using 4 spaces for new lines.
```

Comment:
according to the source, they should be rendered like this 
![image](https://github.com/Lundalogik/lime-elements/assets/35954987/b6cdd3bb-bd58-483c-8034-9e41603842c5)
But not sure how to test and see them. 🤔 maybe we should just remove this example.

Reply:
I have got footnotes working @Kiarokh 🥳

Reply:
Wonderful! <3